test/emails/UserCreatedNotification.test.tsx

```diff
@@ -0,0 +1,57 @@
+import UserCreatedNotification from "emails/UserCreatedNotification"
+import UserDetails from "types/UserDetails"
+
+describe("WHEN UserCreatedNotification is created", () => {
+  describe("AND valid arguments are passed through", () => {
+    let validUser: UserDetails
+    beforeEach(() => {
+      // Given
+      validUser = {
+        username: "testUser",
+        forenames: "testForename",
+        surname: "testSurname",
+        emailAddress: "test@test.com",
+        endorsedBy: "testEndorser",
+        groups: [{ id: "1", name: "B7Allocator", friendly_name: "Allocator" }],
+        orgServes: "testOrg"
+      }
+    })
+
+    it("SHOULD contain all necessary information", () => {
+      // When
+      const result = UserCreatedNotification({ user: validUser })
+      // Then
+      expect(result.text).toBeTruthy()
```

Comment:
can we do a matchsnapshot command? might be easier?

Reply:
I prefer this approach because this is not testing for the formatting of the component; which is where a snapshot test shines. 

I am much more interested in the content of the email.

You can argue that a snapshot does that implicitly but this explicit test is capturing the requirements from the ticket and can be updated as requirements evolve.

Reply:
you make a good point, fair enough

Reply:
If we do this elsewhere (use snapshots to capture content of a component given a number of inputs) we should be wary because a snapshot test will not protect us from mistaken changes ;(

Reply:
nah, it was mostly for content. I was considering that you took great care in ensuring that the format is theer. Hence why I was recommending it

Reply:
I suppose we need to bare in mind that this 'component' just returns an object with text, html, and subject fields. 

To do this I would need to embed this in a test component to snapshot.